### ISSUE TITLE
wrong accuracy_score input ?

### ISSUE THREAD

**Issue by Karlheinzniebuhr** [Wensi-Tang/OS-CNN#12]
Hi Wensi, I was extending the source code of os-cnn when this caught my attention. It looks to me like the order of predictions and labels isn't correct? It should be _accuracy_score(y_true, y_pred)_ but it seems to be _accuracy_score(y_pred, y_true)_

![image](https://user-images.githubusercontent.com/5278928/171065626-5fe8ab9c-6c4f-4437-82d0-0a7e9a9da3c2.png)

**Comment by Wensi-Tang** [comment]
The order will not influence the result of this function. 
Check Documentation [here](https://scikit-learn.org/stable/modules/model_evaluation.html#accuracy-score) 
or
Check it with code
```Python
from sklearn.metrics import accuracy_score as accuracy_score
import numpy as np

for i in range(5):
    arr_1 = np.random.randint(2,size = 100)
    arr_2 = np.random.randint(2,size = 100)
    print(accuracy_score(arr_2,arr_1),accuracy_score(arr_1,arr_2))

for i in range(5):
    arr_1 = np.random.randint(3,size = 100)
    arr_2 = np.random.randint(3,size = 100)
    print(accuracy_score(arr_2,arr_1),accuracy_score(arr_1,arr_2))
```

**Comment by Karlheinzniebuhr** [comment]
thanks for clarifying got a little worried😅
